examples/immutable-values/immutable_values.description

```diff
@@ -0,0 +1,12 @@
+// Ballerina allows making values immutable or read-only.
+//
+// A value of type `anydata` can be made immutable by calling the `.freeze()` built-in method on it. A successful
+// freeze attempt would return the value with the frozen flag set. Calling freeze on structures would recursively
```

Comment:
Can we simplify this and say "calls itself" or something instead of "recursively"? 

I feel a developer might understand the term but there might be a point in time when we decide to localize some of this content or some users might use Google Translate. If it's too complex, the translation wouldn't be very clear. Same comment applies to "immutable". Can we rethink the wording?

Reply:
I've updated `recursively` to `deeply`. 

As for `immutable`, IMO, it would be better if we could keep it since it is the core concept we are trying to demonstrate here. Since in the first sentence, we've also included a more familiar term `read-only`, shall we keep immutable as is? WDYT?